gitlab/resource_gitlab_project.go

```diff
@@ -162,6 +162,11 @@ var resourceGitLabProjectSchema = map[string]*schema.Schema{
 		Type:     schema.TypeBool,
 		Optional: true,
 	},
+	"squash_option": {
+		Type:     schema.TypeString,
```

Comment:
Plase use validation.StringInSlice.

Reply:
Mmh validation with stringToSquashOptionValue would be even better.

Reply:
It's kind of hard to pass the type of stringToSquashOptionValue function into ValidateFunc :/. So I stayed with validation.StringInSlice like in other examples.